frontend/server/src/DAO/ProblemsetAccessLog.php

```diff
@@ -39,7 +39,8 @@ public static function GetAccessForProblemset(int $problemsetId) {
                                 1
                         ),
                         "user-rank-unranked"
-                    ) `classname`
+                    ) `classname`,
+                    "open" AS eventType
```

Comment:
```suggestion
                    "open" AS event_type
```

todo lo SQLoso va en snake_case. igual en los demás queries.

Reply:
Listo